src/CFamily.UnitTests/CMake/CMakeSettingsProviderTests.cs

```diff
@@ -66,71 +61,64 @@ public void Find_FileExists_ParsedSettingsReturned()
                 }
             };
 
-            var fileSystem = new Mock<IFileSystem>();
-            fileSystem.SetupCMakeSettingsFileExists(cmakeSettingsLocation, cmakeSettings);
+            var cmakeSettingsFile = Path.Combine(RootDirectory, CMakeSettingsProvider.CMakeSettingsFileName);
+            var cmakeListsFile = Path.Combine(RootDirectory, CMakeSettingsProvider.CMakeListsFileName);
```

Comment:
Do we have any tests that check that `CMakeSettingsProvider.CMakeSettingsFileName` and `CMakeListsFileName` have the values we expect?

Reply:
Yes, Find_FileExists_ParsedSettingsReturned